deployer/models/service.go

```diff
@@ -596,6 +601,22 @@ func (service *Service) UpdateHealthy(asgc aws.ASGAPI, elbc aws.ELBAPI, albc aws
 	return nil
 }
 
+func (service *Service) SlowStartDuration(albc aws.ALBAPI) int {
+	longest := 0
+	for _, arn := range service.Resources.TargetGroups {
+		output, _ := albc.DescribeTargetGroupAttributes(&elbv2.DescribeTargetGroupAttributesInput{TargetGroupArn: arn})
+		for _, attribute := range output.Attributes {
+			if *attribute.Key == "slow_start.duration_seconds" {
```

Comment:
ensure attribute.Key and Value are not nil

Reply:
I dont think they can be nil, they are string pointers?